test/Microsoft.ML.Benchmarks/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Toolchains.CsProj;
+using BenchmarkDotNet.Toolchains.InProcess;
+using System;
+using System.IO;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+using Microsoft.ML.Benchmarks;
+
+namespace Microsoft.ML.Benchmarks
+{
+    class Program
```

Comment:
>Program [](start = 10, length = 7)

Whis is this an EXE and not a test? 
I wonder would it be easier if we just have such tests as regular unit tests?  from a dev perpective, we have nice tools to run and compare results for tests.

Reply:
BenchmarkDotNet (BDN) test are exes. I am pretty sure they cannot be dlls that are run as part of unit tests. I think @adamsitnik is working on infrastructure that will let us run the tests in the outer loop (daily runs)